docs/src/formula.md

```diff
@@ -14,167 +14,184 @@ fields with possibly heterogeneous types.  One of the primary goals of
 `StatsModels` is to make it simpler to transform tabular data into matrix format
 suitable for statistical modeling.
 
-At the moment, "tabular data" means an `AbstractDataFrame`.  Ultimately, the
-goal is to support any tabular data format that adheres to a minimal API,
-**regardless of backend**.
-
-## The `Formula` type
-
-The basic conceptual tool for this is the `Formula`, which has a left side and a
-right side, separated by `~`. Formulas are constructed using the `@formula` macro:
-
-```jldoctest
-julia> @formula(y ~ 1 + a)
-Formula: y ~ 1 + a
+At the moment, "tabular data" means a
+[Tables.jl](https://github.com/JuliaData/Tables.jl) table, which will be
+materialized as a `Tables.ColumnTable` (a `NamedTuple` of column vectors).  Work
+on first-class support for streaming/row-oriented tables is ongoing.
+
+## The `@formula` DSL
+
+StatsModels implements the [`@formula`](@ref) domain-specific language (DSL) for
+describing table-to-matrix transformations.  This DSL is designed to be familiar
+to users of other statistical software, while also taking advantage of Julia's
+unique strengths to be fast and flexible.
+
+A basic formula is composed of individual *terms*—symbols which refer to data
```

Comment:
Add spaces around dashes here and below? I first read this as "terms-symbols". Or maybe change the syntax.

Reply:
hard to tell with the monospaced font but those are em dashes so they _should_ be sufficiently visually distinct (unless of course you've built the docs and looked there, in which case I'm willing to reconsider :))

Reply:
I guess that's a difference between English and French typography rules...